### ISSUE TITLE
Can you provide the cromwell images?

### ISSUE THREAD

**Issue by liuxinf** [IBM/cromwell-on-ibmcloud#2]
Can you provide the cromwell images?

**Comment by liuxinf** [comment]
This is a good solution.but compilation is a little difficult for me

**Comment by takeshi-yoshimura** [comment]
Please use icr.io/cromwell-on-ibmcloud/cromwell:78-k8s in https://github.com/IBM/cromwell-on-ibmcloud/blob/main/k8s/cromwell.yaml

**Comment by liuxinf** [comment]
Can I use this backend directly on k8s, without the ibm cloud? In this case, I won't be able to use the image on ibm




---Original---
From: "Takeshi ***@***.***&gt;
Date: Mon, Oct 24, 2022 10:04 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [IBM/cromwell-on-ibmcloud] Can you provide the cromwell images?(Issue #2)




 
Please use icr.io/cromwell-on-ibmcloud/cromwell:78-k8s in https://github.com/IBM/cromwell-on-ibmcloud/blob/main/k8s/cromwell.yaml

**Comment by liuxinf** [comment]
thanks



---Original---
From: "Takeshi ***@***.***&gt;
Date: Mon, Oct 24, 2022 10:04 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [IBM/cromwell-on-ibmcloud] Can you provide the cromwell images?(Issue #2)




 
Please use icr.io/cromwell-on-ibmcloud/cromwell:78-k8s in https://github.com/IBM/cromwell-on-ibmcloud/blob/main/k8s/cromwell.yaml

**Comment by takeshi-yoshimura** [comment]
Yes, you can directly use this in your Kubernetes environment even if it is not in IBM Cloud.

**Comment by liuxinf** [comment]
thank you very much



---Original---
From: "Takeshi ***@***.***&gt;
Date: Mon, Oct 24, 2022 10:59 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [IBM/cromwell-on-ibmcloud] Can you provide the cromwell images?(Issue #2)




 
Yes, you can directly use this in your Kubernetes environment even if it is not in IBM Cloud.

**Comment by liuxinf** [comment]
Hello，can the backend of k8s be used in actual production?Because I lack some real cases.I want to confirm with you.thank you.



---Original---
From: "Takeshi ***@***.***&gt;
Date: Mon, Oct 24, 2022 10:59 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [IBM/cromwell-on-ibmcloud] Can you provide the cromwell images?(Issue #2)




 
Yes, you can directly use this in your Kubernetes environment even if it is not in IBM Cloud.

**Comment by liuxinf** [comment]
Hello.excuse me.If whether the external interfaces provided by the k8s-backend-78 cromwrll can correctly obtain data when interconnecting with k8s backend.such as job status,logs,historyical query,and indicator.thank you



---Original---
From: "Takeshi ***@***.***&gt;
Date: Mon, Oct 24, 2022 10:59 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [IBM/cromwell-on-ibmcloud] Can you provide the cromwell images?(Issue #2)




 
Yes, you can directly use this in your Kubernetes environment even if it is not in IBM Cloud.

**Comment by takeshi-yoshimura** [comment]
@liuxinf 
please be aware of the fact that this k8s backend is currently experimental i.e., not for production and not merged to upstream Cromwell.

As for data, you need to set up NFS or other distributed filesystems to share data among jobs in Kubernetes. if you store data on an S3 bucket, you need to download data from the bucket to the distributed filesystem before starting jobs.
